src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -1857,11 +1860,14 @@ let internal_commands logger =
               ~conf_dir:(Some conf_dir) ()
           in
           let%bind result =
+            let cap lst =
```

Comment:
```suggestion
            let cap =
```

Reply:
this was the easiest way for the type to resolve as an 'a list